content/jsoc/gsoc/flux.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: page
+title: Flux
```

Comment:
Still don't know why this would be called Flux. Half of the projects have nothing to do with Flux. Maybe we should just make a different SciML page.

Reply:
Yes, make a different SciML page.